concent_api/core/models.py

```diff
@@ -1,17 +1,19 @@
 from django.db.models import BinaryField
 from django.db.models import BooleanField
+from django.db.models import CharField
 from django.db.models import DateTimeField
 from django.db.models import ForeignKey
-from django.db.models import IntegerField
 from django.db.models import PositiveSmallIntegerField
 from django.db.models import Model
 
+from .constants       import MESSAGE_TASK_ID_MAX_LENGTH
+
 
 class Message(Model):
     type        = PositiveSmallIntegerField()
     timestamp   = DateTimeField()
     data        = BinaryField()
-    task_id     = IntegerField(null = True)
+    task_id     = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH, blank = True, default = '')
```

Comment:
Please remove the default. The field should be assigned a value explicitly.

Also, now I see that this field should really be `blank = False` and `unique` too.